src/styles/constants.scss

```diff
@@ -0,0 +1,19 @@
+// Colors
+
+$disabledInputColor: #302f2f;
+$errorColor: #eb2a2a;
+$shadowColor: rgba(0, 0, 0, .5);
+$skyblueBackgroundColor: #2fbcf7;
+
+// Fonts
+
+$baseFontSize: 16px;
+$headerFontSize: #{(20px/$baseFontSize)}rem;
+$subheaderFontSize: #{(18px/$baseFontSize)}rem;
+$mediumFontSize: #{(14px/$baseFontSize)}rem;
+$smallFontSize: #{(12px/$baseFontSize)}rem;
+
+$headerFontHeight: #{(27px/20px)};
+$subheaderFontHeight: #{(32px/18px)};
+$mediumFontHeight: #{(26px/14px)};
+$smallFontHeight: #{(16px/12px)};
```

Comment:
What do you think about that? I did it this way because I want to keep the relation with the Figma designs so if the designs will change, I could update the values without doing any calc.
If it's too weird, I change it.
Please, give me your opinion.

Reply:
I recommend implementing something like `rem-calc` from foundation

https://gist.github.com/kylewebdev/f36acbfd2e9257d88253#file-rem-calc-scss-L13